src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketsIds;
+    private Long userId;
```

Comment:
Checklist: When choosing the fields to be returned in your ShoppingCartResponseDto, keep in mind that cartId and userId hold the same value.